### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by vantu5z**
fix some plural forms

### CODE REVIEW

**Comment by sam-m888** [comment]
Your latest commit 62e27370e failed on travis-ci?

**Comment by Paul-Franklin** [comment]
What does check_po say?

(A very quick visual glance showed at least one mismatch
of Python curly-brace argument names.)

**Comment by Paul-Franklin** [comment]
The msgid "%(str1)s in %(str2)s. " does not match the
msgstr "%(part1)s в %(part2)s" but since I don't have
JavaScript enabled I can't view the exact complaint:
https://travis-ci.org/gramps-project/gramps/builds/95128760

**Comment by Paul-Franklin** [comment]
I have committed your 1561098 po/ru.po to gramps42 and master,
but I changed its PO date to 7 December to make it unique (since
the two previous commits in your private repo had the same date).
Thank you for helping make gramps better.  

https://github.com/gramps-project/gramps/commit/dba52f4d2f9a45998e7d1dc4a0b12c790842bb78
https://github.com/gramps-project/gramps/commit/f4f5544060de3c4432200cd40983a3f398454476

File: po/ru.po [hunk 1]
```diff
@@ -22863,9 +22860,8 @@ msgid "Illness"
 msgstr "Болезнь"
 
 #: ../gramps/plugins/importer/importgeneweb.py:109
-#, fuzzy
 msgid "List Passenger"
-msgstr "Уместить на страницу"
+msgstr "Список рассажиров"
```
Comment (baznikin) [inline po/ru.po:22864]: typo

**Pull request closed without merging**